cmd/helm/init.go

```diff
@@ -284,6 +284,9 @@ func (i *initCmd) run() error {
 				}
 				fmt.Fprintln(i.out, "\nTiller (the Helm server-side component) has been upgraded to the current version.")
 			} else {
+				if settings.Debug {
+					fmt.Fprintln(i.out, fmt.Sprintf("[INFO] The error received while trying to init: %s", err))
+				}
```

Comment:
I like the desire to display the debug information. But, can you use the `debug` function to do this instead? It's part of the `main` package here in `printer.go`. It has the if statement wrapper, etc.

Reply:
@mattfarina Updated the logic to use `debug` function instead